helix-core/src/syntax.rs

```diff
@@ -92,7 +89,7 @@ pub struct LanguageConfiguration {
     pub(crate) highlight_config: OnceCell<Option<Arc<HighlightConfiguration>>>,
     // tags_config OnceCell<> https://github.com/tree-sitter/tree-sitter/pull/583
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub language_server: Option<LanguageServerConfiguration>,
+    pub language_servers: Option<Vec<LanguageServerConfiguration>>,
```

Comment:
Quick note: just use `Vec<T>` rather than `Option<Vec<T>>`. An empty vector is the same as `None`.

Reply:
You're right, kinda went on "dumb" replace mode I guess.

The code is definitely less convoluted this way...